backend/src/server.js

```diff
@@ -280,7 +284,8 @@ async function main() {
       "user succesfully logged in with twitter",
       user.screen_name
     );
-    req.session.save(() => res.redirect("http://localhost:3000/"));
+    // NOTE: it might be fine to just redirect to '/' for all cases? hostname is inferred
```

Comment:
The frontend and backend hosts are different. back  is localhost:4000, front is localhost:3000. So in that case it wouldn't be inferred. Should it not be that way?

Reply:
updated! removed the comment after you clarified 21f9135beb779a44b80d3d47f1be8a4f45376420